samples/vbusin/src/main.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <npmx_driver.h>
+#include <npmx_vbusin.h>
+
+#define LOG_MODULE_NAME vbusin
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+/**
+ * @brief Callback function to be used when VBUSIN VOLTAGE event occurs.
+ *
+ * @param[in] p_pm Pointer to the instance of PMIC device.
+ * @param[in] type Callback type. Should be @ref NPMX_CALLBACK_TYPE_EVENT_VBUSIN_VOLTAGE.
+ * @param[in] mask Received event interrupt mask.
+ */
+void vbusin_voltage_callback(npmx_instance_t *p_pm, npmx_callback_type_t type, uint8_t mask)
+{
+	/* Task current apply have to be triggered each time when USB is (re)connected. */
```

Comment:
```suggestion
	/* Task current apply have to be triggered each time when USB is (re)connected. */
```
Could you please clarify the first part of the sentence? We have two verbs one after another, **apply** and **have to**.

Reply:
Changed to `Current limit have to be applied each time when USB is (re)connected.`.